process/project_proposals.adoc

```diff
@@ -73,7 +84,8 @@ image::incubation-process.png[Incubation process]
    * The proposal addresses how the project has grown since incubation and any concerns from incubation DD in addition to the standard graduation requirements.
    * Projects will be reviewed on a rolling basis as they apply, instead of two meetings a year.    
 . * If a TOC member steps forward to support the project as a sponsor and determines the Graduation DD document is finalized, the TOC member kicks off two week period of time for public comment on the TOC mailing list
-   * The email should contain a link to the proposal pull request and incubation DD document.
+   * Lobbying for a sponsor is not permitted.
```

Comment:
Perhaps we could elaborate on what _lobbying_ means in this context.  We want to be inclusive and we don't want to discourage projects from participating in the TOC meetings and reaching out to the TAGs for advise (or scheduling presentations/discussions).

My two cents.